### PR TITLE
Mak/landing page

### PR DESCRIPTION
**Pull request by makrandgupta**
Updated design for landing page

image

### CODE REVIEW

**Comment by traversng** [comment]
@makrandgupta Add screenshot 🙏

File: src/app/page.tsx [hunk 1]
```diff
@@ -26,11 +27,20 @@ export default function Home() {
 
   const ctaButton = useMemo(() => {
     if (isClient) {
-      return isConnected ? (
+      return isConnected || true ? (
```
Comment (traversng) [inline src/app/page.tsx:30]: wont this always return true?

File: src/app/page.tsx [hunk 2]
```diff
@@ -48,36 +58,58 @@ export default function Home() {
...
-            align="center"
-            color={Color.ForegroundMuted}
-          >
-            Collect onchain art across Miami and exchange it for exclusive
-            merch.
-          </Typography>
-        </Stack>
-        <Stack paddingTop={5} alignItems="center">
+      <Stack
+        component="div"
```
Comment (siwan-cb) [inline src/app/page.tsx:62]: you don't need this

File: src/app/page.tsx (hunk 1 above)
Comment (makrandgupta) [reply src/app/page.tsx:30]: Ah yes, I put that in to force the other option to show up, will remove

**Pull request merged**